scripts/gen_aac.py

```diff
@@ -168,30 +180,117 @@ def generate(self, download, jobs):
+                    absolute_path = os.path.join(os.getcwd(), dest_dir, name + ".mp4")
+                    absolute_path_folder = os.path.join(os.getcwd(), dest_dir)
+                    
+                    if os.path.exists(absolute_path):
+                        os.remove(absolute_path)
+                        os.rmdir(absolute_path_folder)
+                    else:
+                        raise Exception(f"The file {absolute_path} couldn't be deleted")
+
+                    # Remove test vectors from test suite and the corresponding links
+                    del(test_suite.test_vectors[str(name)])
+
+                    # Rewrite compressed bitstream link list
+                    compressed_bitstream_links[:] = [link for link in compressed_bitstream_links if link.split("/")[-1].split(".")[0] != name]
+
         compressed_bitstream_names = [os.path.splitext(os.path.basename(x))[0] for x in compressed_bitstream_links]
 
         with urllib.request.urlopen(self.url_reference_vectors) as resp:
             data = str(resp.read())
-            hparser.feed(data)
-        raw_bitstream_links = [url for url in hparser.links if url.endswith(tuple(RAW_EXTS))]
+            hparser_raw.feed(data)
+        raw_bitstream_links = [
+            url for url in hparser_raw.links if url.endswith(tuple(RAW_EXTS))
+        ]
+
+        # The reference files are divided in two DVDs for MPEG4_AAC-MP4 test suite
+        if test_suite.name == "MPEG4_AAC-MP4":
+            hparser_raw_extra = HREFParser()
+            
+            # Get the DVD3 wav files
+            with urllib.request.urlopen(URL_MPEG4_WAV_REFS_DVD3) as resp:
+                data = str(resp.read())
+                hparser_raw_extra.feed(data)
+            
+            raw_bitstream_links_MP4 = [
+                url for url in hparser_raw_extra.links if url.endswith(tuple(RAW_EXTS))
+            ]
+
+            # Adding the DVD3 wav files to the rest of the files
+            raw_bitstream_links = raw_bitstream_links + raw_bitstream_links_MP4
 
         raw_bitstream_names = [
             os.path.splitext(os.path.basename(x))[0].split('_f')[0] for x in raw_bitstream_links
         ]
 
-        missing_files = [x for x in set(compressed_bitstream_names).difference(raw_bitstream_names)]
+        missing_files = [
+            x for x in set(compressed_bitstream_names).difference(raw_bitstream_names)
+        ]
         if missing_files:
-            print(f"Missing reference files: {missing_files}")
+            #print(f"Missing reference files: {missing_files}")
             for missing_file in missing_files:
                 print(f"Skipping test vector {missing_file}, as the reference file is missing.")
 
-        raw_bitstream_names = [name for name in compressed_bitstream_names if name not in missing_files]
+        raw_bitstream_names = [
+            name for name in compressed_bitstream_names if name not in missing_files
+        ]
```

Comment:
I think this happened because pylint is set to 80 instead of 120 characters per line in your config. Can you check it out and revert.
If `make check` passes with the previous syntax you should be fine